Makefile

```diff
@@ -168,18 +170,20 @@ cache/$(EXTENSION_FUNCTIONS):
 
 ## sqlite-src
 .PHONY: sqlite-src
-sqlite-src: sqlite-src/$(SQLITE_AMALGAMATION) sqlite-src/$(EXTENSION_FUNCTIONS)
+sqlite-src: sqlite-src/$(SQLITE_AMALGAMATION) sqlite-src/$(SQLITE_AMALGAMATION)/$(EXTENSION_FUNCTIONS)
```

Comment:
I think this was a bug previously, that it depended upon `src//$(EXTENSION_FUNCTIONS)` and not `src/$(SQLITE_AMALGAMATION)/$(EXTENSION_FUNCTIONS)`.

I think it was never noticed because we don't ever manually build the `sqlite-src` target